Fixed #37045 -- renamed `savepoint` to `savepoint_create` (#21119)
django/django: docs/releases/6.1.txt

  :class:`~django.db.models.BitOr`, and :class:`~django.db.models.BitXor`
  classes.

> * ``django.db.transaction.savepoint`` has been deprecated in favor of
>   ``django.db.transaction.savepoint_create``.

Reviewer: ```suggestion
* ``django.db.transaction.savepoint()`` is deprecated in favor of
  ``django.db.transaction.savepoint_create()``.
```
